components/ToggleButton/ToggleButton.js

```diff
@@ -1,14 +1,35 @@
 import React from 'react'
 import s from './ToggleButton.css'
 
-const ToggleButton = (props) => (
-  <button
-    className={`${s.toggleButton} ${props.enabled ? s.enabled : ''} ${props.extraClasses}`}
-    onClick={props.onClick}
-    title={props.label}
-  >
-  {props.label}
-  </button>
-)
+// A button with toggle classes based on state and
+// either a visual-only display (for putting inside of an a link)
+// or onClick support (which will render as a button)
+const ToggleButton = (props) => {
+  const className = `${s.toggleButton}
+                     ${props.enabled ? s.enabled : ''}
+                     ${props.extraClasses || ''}`;
```

Comment:
Got rid of a tiny bug where this would generate class name ending with a class `.undefined`